apps/web/components/ui/Dropdown.tsx

```diff
@@ -25,8 +25,11 @@ export const DropdownMenuContent = forwardRef<HTMLDivElement, DropdownMenuConten
   ({ children, ...props }, forwardedRef) => {
     return (
       <DropdownMenuPrimitive.Content
+        portalled={false}
         {...props}
-        className="z-10 mt-1 w-48 origin-top-right rounded-sm bg-white text-sm shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none"
+        className={`${
+          props.portalled ? `` : `md:-ml-[55px]`
+        } z-10 mt-1 -ml-0 w-48 origin-top-right rounded-sm bg-white text-sm shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none lg:-ml-56`}
```

Comment:
As there a quite a few views with different toolsets, i.e. full sidebar, collapsed sidebar and no sidebar (bottom bar), I needed to take that into consideration. These tweaks are not needed when the dropdown is "portalled".